routers/web/repo/commit.go

```diff
@@ -390,12 +390,6 @@ func Diff(ctx *context.Context) {
 		}
 	}
 
-	ctx.Data["BranchName"], err = commit.GetBranchName()
```

Comment:
This branch name was used for "cherry pick" dropdown, but it doesn't make sense to fill current branch into the dropdown (why cherry-pick to the current branch?). So it could be completely removed.